src/main/java/org/usfirst/frc4048/common/swervev3/vision/FilterResult.java

```diff
@@ -0,0 +1,5 @@
+package org.usfirst.frc4048.common.swervev3.vision;
+
+public enum FilterResult {
```

Comment:
This can probably move to the interface

Reply:
I prefer separate files for enums, classes, interfaces, etc.
Is there a reason you want to move it to the interface?

Reply:
Normally, if an enum is tightly coupled to a particular implementation, it makes it easier to follow the code if it is all in the same class.